DeepSpeech.ipynb

```diff
@@ -106,7 +107,14 @@
     "do_fulltrace = bool(len(os.environ.get('ds_do_fulltrace', '')))\n",
     "\n",
     "if do_fulltrace:\n",
-    "    check_cupti()"
+    "    check_cupti()\n",
+    "\n",
+    "track_gpu_usage = bool(len(os.environ.get('ds_track_gpu_usage', '')))\n",
+    "gpu_usage_charts = os.environ.get('ds_gpu_usage_charts', 'gpu_usage_%%s_%d.png' % int(time.time()))\n",
+    "\n",
+    "if track_gpu_usage:\n",
+    "    gu = GPUUsage()\n",
+    "    gu.start()"
```

Comment:
Sure, but again, this is to deal with interacting with Tensorflow's flow itself, while our thread here is completely separate to that process. Moving the code there makes it less readable, hence why I am a bit relunctant except if there is a solid reason :)